spec/ics-004-channel-and-packet-semantics/README.md

```diff
@@ -599,41 +604,75 @@ function recvPacket(
 
     // all assertions passed (except sequence check), we can alter state
 
-    // always set the acknowledgement so that it can be verified on the other side
-    provableStore.set(
-      packetAcknowledgementPath(packet.destPort, packet.destChannel, packet.sequence),
-      hash(acknowledgement)
-    )
-
     if (channel.order === ORDERED) {
       nextSequenceRecv = provableStore.get(nextSequenceRecvPath(packet.destPort, packet.destChannel))
       abortTransactionUnless(packet.sequence === nextSequenceRecv)
       nextSequenceRecv = nextSequenceRecv + 1
       provableStore.set(nextSequenceRecvPath(packet.destPort, packet.destChannel), nextSequenceRecv)
-    } else
-      abortTransactionUnless(provableStore.get(packetAcknowledgementPath(packet.destPort, packet.destChannel, packet.sequence) === null))
+    } else {
+      // for unordered channels we must set the receipt so it can be verified on the other side
```

Comment:
I know this isn't needed for correctness since it is implicit once you understand the spec, but I don't see any references saying the receipt is just a bit and doesn't contain any other information. I could see a new reader thinking the receipt should be provided by the calling module

Reply:
Aye, added a comment.